interface/patient_file/summary/demographics.php

```diff
@@ -1762,34 +1746,53 @@ function setMyPatient() {
 }
 
 if ($picture_url) {
-    $imageUrl = "../../../sites/".$_SESSION['site_id']."/profile_pictures/".$picture_url;
-    $successMsg = xl("Profile picture has been set successfully");
-    $errorMsg = xl("Error in setting the profile picture");
   //show success message and update the value in db and also refresh the page.
   if (sqlQ("UPDATE patient_data SET picture_url='$picture_url' WHERE pid='$pid'")) {
         echo "<script>
-    iziToast.success({
-        title: 'User Profile Updated',
-        icon: 'fa fa-picture-o',
-        message: '$successMsg',
-        onOpening: function () {
-          $('#prof_img').attr('src', '$imageUrl');
-      }
-    });
+    if (confirm('profile picture has been set successfully')) {
+      window.location = 'demographics.php?set_pid=$pid';
+    }
+    else {
+       window.location = 'demographics.php?set_pid=$pid';
+    }
     </script>";
   }
   else {
-      echo "<script>
-            iziToast.error({
-                title: 'Error',
-                icon: 'fa fa-warning',
-                message: '$errorMsg',
-            });
+            echo "<script>
+    if (confirm('Error in setting the profile picture')) {
+      window.location = 'demographics.php?set_pid=$pid';
+    }
+    else {
+       window.location = 'demographics.php?set_pid=$pid';
+    }
     </script>";
   }
 }
 else {
   //show failure message.
 }
 
+//image mime check with all image formats.
+function has_right_mime($image_properties) {
+  $mime = $image_properties["mime"];
+    $mime_types = array('image/png',
+                            'image/jpeg',
+                            'image/gif',
+                            'image/bmp',
+                            'image/vnd.microsoft.icon');
+   
+   return in_array($mime, $mime_types);
+}
+
+//image file extension change
+function has_right_extension($image_file_type, $extensions) {
+  
+  return in_array($image_file_type, $extensions);
+}
+
+//image size check
+function has_right_size($size) {
+  return $size < 20971520;
+}
+
```

Comment:
Should these be moved to a utility file somewhere?

Reply:
@robbyoconnor Just mention the file and I will move it D: